style.css

```diff
@@ -1,1242 +1,1032 @@
+.intro-text h1 {
+  font-family: "Crete Round", serif;
+  font-size: 56px;
   line-height: 64px;
   letter-spacing: 0.37px;
   width: 20rem;
   color: #ff6b00;
+  margin-left: 0;
+  margin-top: 0;
+  animation: colorC 5s;
 }
 
-.profession h2 {
-  font-family: 'Roboto', serif;
-  font-weight: 700;
-  font-size: 24px;
-  color: #fff5e1;
+.portfolio-social a {
+  color: #dbd8d7;
 }
 
-.discription p {
-  font-weight: 400;
-  font-size: 18px;
-  line-height: 24px;
-  color: #fff;
-  width: 21rem;
-  font-family: 'Poppins', sans-serif;
+.portfolio-img {
+  position: absolute;
+  bottom: -4rem;
+  right: 0;
 }
 
-.social-links {
+.navbar {
   display: flex;
-  width: 10rem;
   justify-content: space-between;
-  align-items: center;
-  list-style: none;
-  gap: 0;
+  background-color: #3c3a39;
+  color: #fff;
+  font-weight: 700;
+  font-size: 1.8rem;
+  padding: 1rem 3rem;
 }
 
-.links {
-  padding: 15px;
-  display: flex;
-  width: 10rem;
-  justify-content: space-between;
-  align-items: center;
+.navbar-list {
   list-style: none;
-  gap: 0 rem;
-  position: relative;
-  top: 5180px;
-  left: 90px;
+  display: flex;
 }
 
-.divider-2 {
-  position: relative;
-  height: 1px;
-  top: 5180px;
-  width: 410px;
-  background: #dfe1e6;
+.navbar-link {
+  display: none;
 }
 
-.my-recent-works {
-  display: grid;
-  grid-template-columns: auto;
-  align-items: center;
-  padding: 120px 0;
-  gap: 80px;
-  position: absolute;
-  width: 375px;
-  height: 3455px;
-  left: 20px;
-  top: 900px;
+.navbar-list-item a {
+  color: #fff;
 }
 
-.recent-works {
+.project {
+  background-color: #fff;
   display: flex;
   flex-direction: column;
-  align-items: center;
-  padding: 0;
-  gap: 24px;
-  grid-column: 1/ span 2;
-  left: 22px;
-  top: 120px;
-  border-radius: 0;
-  position: absolute;
+  padding: 2rem;
 }
 
-.header-content {
-  font-family: 'Crete Round', serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 40px;
-  line-height: 52px;
-  display: flex;
-  align-items: center;
-  color: #1c1a19;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
+.underlined {
+  border-bottom: 1px solid #212121;
 }
 
-.hline {
-  width: 327px;
-  height: 0;
-  border: 1px solid #212121;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
+.underlined-hr {
+  display: none;
 }
 
-.recent-work-container {
-  left: 0;
-  top: 307px;
-  border-radius: 0;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 0;
-  gap: 24px;
-  width: 375px;
-  height: 3028px;
-  flex: none;
-  flex-grow: 0;
-  position: absolute;
+.projects-heading1 h3 {
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  color: #fff;
+  text-align: left;
+  font-weight: 400;
+  margin-top: 5rem;
+  margin-left: 1rem;
 }
 
-/* below are css for the featured img section */
+.project .underlined {
+  color: #1d1a19;
+  font-size: 40px;
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+  margin: 3rem 0;
+  text-align: center;
+  padding: 2rem;
+}
 
-.featured-image {

```

Comment:
- [ ] Please you have to make your project still looks great on a mobile design by making it responsive. :pray: Currently, you have a horizontal scroll on the mobile view which means your project is not responsive on the mobile view so make sure that you remove the horizontal overflow on the mobile view.

  [Screencast from 03-28-2023 09:24:00 PM.webm](https://user-images.githubusercontent.com/92303891/228372196-4783301a-85dc-416b-b6b4-d328323e982f.webm)

- [ ] Please the card here is overlapping on mobile view so make sure that you fix it.

  ![Screenshot from 2023-03-28 21-32-48](https://user-images.githubusercontent.com/92303891/228372264-f0bb90ac-edc9-4210-8b03-b988feb874ed.png)